doc/cli/npm-install.md

```diff
@@ -14,7 +14,7 @@ npm-install(1) -- Install a package
     npm install <tarball url>
     npm install <folder>
 
-    alias: npm i
+    aliases: npm i, npm add
```

Comment:
oof, i didn't realize this was an alias. it seems like `add` should force `--save`, and it doesn't :-/

Reply:
I've just learnt it. I was trying to do `npm adduser` but I did `npm add user` 😀.
Just trying `ng add`, its seems to be forcing `--save`.

Reply:
if you have `save=false` in your `.npmrc`, does it still force `--save`?

Reply:
No, if I have `save=false` in my `.npmrc` (whether it's local to my project or global), it doesn't force `--save`.